e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -396,6 +396,31 @@ describe('Customization', () => {
             expect(config.TeamSettings.EnableCustomBrand).to.equal(false);
         });
     });
+
+    it('T1025 - Site Name - Main Menu ➜ About and About Modal show custom name', () => {
```

Comment:
```suggestion
    it('MM-T1025 - Site Name - Main Menu ➜ About and About Modal show custom name', () => {
```

Reply:
I'll wait for https://github.com/mattermost/mattermost-webapp/pull/6375 to be merged so the file and test surroundings exist already and I can just move the test case over instead of recreating everything (with this and the rest of the tests for the ticket)